### ISSUE TITLE
about attribute depth

### ISSUE THREAD

**Issue by yjy4231** [SPengLiang/DID-M3D#25]
Sorry to disturb you again!
About the  attribute depth in the  `kitti.py`, `att_depth[i] = depth[i] - vis_depth[i]`, vis_depth[i] is from the dense depth map after the affine transformation, but the depth[i] seems to be from the label before the affine transformation. I think the solution of the vis_depth[i] should use the vis_depth[i]  from the dense depth map before the affine transformation. Otherwise, the attribute depth changes before and after the affine transformation. I don't know if I understand correctly. Could you explain it for me， thank you！

**Comment by SPengLiang** [comment]
You are right, vis_depth[i] should denote the orignal dense depth map value. We perform the affine transformation on  the dense depth map to fit the new image shape, but do not scale its value in the dataloader(the scale factor is at https://github.com/SPengLiang/DID-M3D/blob/2475af69c32b5bd5a716aa14f0c6570fa11a640b/lib/models/DID.py#L175 and is used in https://github.com/SPengLiang/DID-M3D/blob/2475af69c32b5bd5a716aa14f0c6570fa11a640b/lib/models/DID.py#L182). 

In other words, the depth value at the corresponding position does not change in the dataloader. Therefore, the attribute depth does not change before and after the affine transformation.
